### PR TITLE
style: minor improvements for source test and risedev

### PR DESCRIPTION
**Pull request by xxchan**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
trivial part picked from  #16244

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: src/risedevtool/src/risedev_env.rs [hunk 1]
```diff
@@ -25,50 +25,53 @@ use crate::{add_hummock_backend, HummockInMemoryStrategy, ServiceConfig};
...
-            break;
-        }
-    }
-    for item in services {
-        if let ServiceConfig::Frontend(c) = item {
-            let listen_address = &c.listen_address;
-            writeln!(env, "RW_FRONTEND_LISTEN_ADDRESS=\"{listen_address}\"",).unwrap();
-            let port = &c.port;
-            writeln!(env, "RW_FRONTEND_PORT=\"{port}\"",).unwrap();
-            break;
```
Comment (kwannoel) [inline src/risedevtool/src/risedev_env.rs:71]: Why do we start frontend separately from meta and CN previously? 🤔

**Review by kwannoel — APPROVED** [review]
Rest changes LGTM, just don't understand changes to `risedev_env`.

File: src/risedevtool/src/risedev_env.rs (hunk 1 above)
Comment (xxchan) [reply src/risedevtool/src/risedev_env.rs:71]: I think you added it 🤣 
https://github.com/risingwavelabs/risingwave/commit/19aded3042f8f7d405d38a4b12178cae9fbcf9b3

Comment (kwannoel) [reply src/risedevtool/src/risedev_env.rs:71]: I forgot why I added it as well... 😓

**Pull request merged**